extensions/amp-story/1.0/amp-story.js

```diff
@@ -928,6 +928,14 @@ export class AmpStory extends AMP.BaseElement {
         this.handleConsentExtension_();
 
         this.pages_.forEach((page, index) => {
+          if (this.isPaywallStory_()) {
```

Comment:
These lines could be written in the extension itself, because if it exists, it will call the constructor and all the callback methods, so you don't need to check anything here if you move this to `amp-story-subscriptions.js`

Reply:
big +1

Reply:
Great idea! Done.